docs/data/material/components/stacked-snackbars/stacked-snackbars.md

```diff
@@ -0,0 +1,91 @@
+---
+product: material-ui
+title: Stacked snackbars React component
+components: SnackbarsProvider
+materialDesign: https://material.io/components/snackbars
+waiAria: https://www.w3.org/TR/wai-aria-1.1/#alert
+---
+
+# Stacked snackbars
+
+<p class="description">The <code>SnackbarsProvider</code> component and <code>useSnackbars</code> hook allow you to display multiple stacked snackbars.</p>
+
+{{"component": "modules/components/ComponentLinkHeader.js"}}
+
+:::info
+ℹ️ Note that Material Design doesn't advocate for this behavior, as seen in [their snackbar documentation](https://material.io/components/snackbars#usage).
+However, given this is a common pattern seen in multiple webapps, Material UI allows deisgners and developers to break away from that guideline.
+:::
+
+## Usage
+
+To display multiple snackbars, wrap your application with the `SnackbarsProvider` component.
+That's what will enable you to use the `useSnackbars` hook.
+
+If you're using `ThemeProvider`, wrap `SnackbarsProvider` below it, as follows:
+
+```jsx
+<ThemeProvider>
+  <SnackbarsProvider>
+    <YourApp />
+  </SnackbarsProvider>
+</ThemeProvider>
+```
+
+## Maximum snackbars
+
+The `limit` prop can be passed to `SnackbarsProvider` to limit the maximum number of snackbars displayed at the same time.
+
+{{"demo": "MaximumSnackbars.js", "defaultCodeOpen": false}}
+
+## Basic stacked snackbars
+
+The `message` prop is used to show the snackbar message.
+
+The `action` prop can be used to show content on the right side of the snackbar.
```

Comment:
Rephrasing for active voice, and to be more specific (based on my understanding of this prop).

```suggestion
You can add the `action` prop to display additional content on the right side of the snackbar (e.g. a call to action such as an **OK** or **Undo** button).
```